fax/constrained/constrained_test.py

```diff
@@ -64,29 +71,29 @@ def step(i, opt_state):
                             check_dtypes=False)
 
         h = eq_constraints(get_x(final_params))
-        self.assertAllClose(h, tree_util.tree_map(np.zeros_like, h),
+        self.assertAllClose(h, jax.tree_util.tree_map(np.zeros_like, h),
                             check_dtypes=False)
 
-    @parameterized.parameters(
+    @absl.testing.parameterized.parameters(
         {'method': cga_ecp, 'kwargs': {'max_iter': 1000, 'lr_func': 0.5}},
-        {'method': slsqp_ecp, 'kwargs': {'max_iter': 1000}},)
+        {'method': slsqp_ecp, 'kwargs': {'max_iter': 1000}}, )
     @hypothesis.settings(max_examples=10, deadline=5000.)
     @hypothesis.given(
         hypothesis.extra.numpy.arrays(
             onp.float, (2,),
             elements=hypothesis.strategies.floats(0.1, 1)),
     )
     def test_ecp(self, method, kwargs, v):
-        opt_solution = (1./np.linalg.norm(v))*v
+        opt_solution = (1. / np.linalg.norm(v)) * v
```

Comment:
Just formatting? We don't have strong guidelines for formatting, but should perhaps settle on this to avoid having to much diff to parse

Reply:
only formatting, the reason of the random -> jax.random is that especially during development using jax rng is a big overhead so i have used python's random